src/client/src/components/Item.jsx

```diff
@@ -0,0 +1,136 @@
+import deleteIcon from "../images/delete-icon.svg";
+import PropTypes from "prop-types";
+ 
+
+const ListItem = ({ item, itemToDelete, itemToEdit }) => {
+  const [newName, setNewName] = useState(item.itemName);
+  const [editSaveButtonIcon, setEditSaveButtonText] = useState(editIcon);
+  const [itemWasEdited, setItemWasEdited] = useState(false);
+  const [statusCompleteTime, setStatusCompleteTime] = useState("");
+  const [hideClass, setHideClass] = useState("");
+  const [readOnly, setReadOnly] = useState(true);
+  const [decorateClass, setDecorateClass] = useState("");
+  const [checkBoxCheck, setCheckBoxCheck] = useState(item.status);
+
+  const newStatusHandler = async (e) => {
+    if (e.target.checked) {
+      setDecorateClass("decorate");
+      setCheckBoxCheck(true);
+      const timestampNow = new Date();
+      const timestampNowHours = timestampNow.getHours();
+      timestampNow.setHours(timestampNowHours + 3);
+
+      const timestampNowToDb = timestampNow
+        .toISOString()
+        .slice(0, 19)
+        .replace("T", " ");
+      const timestampNowToReact = timestampNowToDb.slice(10, 19);
+      await updateDoneTimestamp(item.itemId, timestampNowToDb);
+
+      await updateStatusInDb(item.itemId, e.target.checked);
+
+      setStatusCompleteTime(`Done at ${timestampNowToReact}`);
+      setHideClass("");
+      return;
+    }
+    await updateStatusInDb(item.itemId, false);
+    setCheckBoxCheck(false);
+    setDecorateClass("");
+    setHideClass("hide");
+  };
+
+  const editName = async () => {
+    if (editSaveButtonIcon == editIcon) {
+      setReadOnly(false);
+      setEditSaveButtonText(saveIcon);
+      return;
+    }
+    setReadOnly(true);
+    setEditSaveButtonText(editIcon);
+    let itemNewName = await itemToEdit(item.itemId, newName);
+
+    setNewName(itemNewName);
+  };
+
+  useEffect(() => {
+    if (checkBoxCheck) {
+      setDecorateClass("decorate");
+      
+      
+      const timeFromDb = item.doneAt.slice(0, 19).replace("T", " ").slice(10, 19)
+     
+      const timeHours = timeFromDb.slice(0,3)
+      const correctHours = Number(timeHours) + 3
+ 
+   const correctTimeToReact = timeFromDb.replace(timeHours, correctHours.toString())
+ setStatusCompleteTime(`Done at ${correctTimeToReact}`)
+     
+    }
+  }, []);
+
+  return (
+    <li className="list-item flex">
+      {" "}
+      <div className="check-box">
+        <input
+          type="checkBox"
+          checked={checkBoxCheck}
+          onChange={newStatusHandler}
+        />
+      </div>
+      <input
+        className={`list-item-text ${decorateClass}`}
+        onChange={(e) => setNewName(e.target.value)}
+        type="text"
+        readOnly={readOnly}
+        value={newName}
+      />
+      {item.isPokemon ? (
+        <div className={`pokemon-photo-div ${decorateClass}`}>
+          {" "}
+          <img src={item.pokemonData}></img>
+        </div>
+      ) : (
+        <div className="space-div"></div>
+      )}
```

Comment:
having inline "if"s in the component renderer makes it less readable.
export to a function called something like `renderPokemonIfNeeded` which will perform this logic and call it from here like `{renderPokemonIfNeeded()}`